app/assets/javascripts/surveys.js

```diff
@@ -0,0 +1,140 @@
+                 '  <div class="wrapper">' +
+                 '    <h1>Tell us what you think of GOV.UK</h1>' +
+                 '    <p class="right"><a href="#survey-no-thanks" id="survey-no-thanks">No thanks</a></p>' +
+                 '    <p><a href="javascript:void()" id="take-survey" target="_blank">Take the 3 minute survey</a> This will open a short survey on another website</p>' +
+                 '  </div>' +
+                 '</section>';
+
+  var userSurveys = {
+    defaultSurvey: {
+      url: 'https://www.surveymonkey.com/s/2MRDLTW',
+      identifier: 'user_satisfaction_survey',
+      template: TEMPLATE,
+      frequency: 50,
+    },
+    smallSurveys: [],
+
+    init: function() {
+      var activeSurvey = userSurveys.getActiveSurvey(userSurveys.defaultSurvey, userSurveys.smallSurveys);
+      if (userSurveys.isSurveyToBeDisplayed(activeSurvey)) {
+        userSurveys.displaySurvey(activeSurvey);
+      }
+    },
+
+    getActiveSurvey: function(defaultSurvey, smallSurveys) {
+      var activeSurvey = defaultSurvey;
+
+      $.each(smallSurveys, function(_index, survey) {
+        if(userSurveys.currentTime() >= survey.startTime && userSurveys.currentTime() <= survey.endTime) {
+          activeSurvey = survey;
+        }
+      });
+
+      return activeSurvey;
+    },
+
+    displaySurvey: function(survey) {
+      $("#user-satisfaction-survey-container").append(survey.template);
+      userSurveys.setEventHandlers(survey);
+
+      var $surveyLink = $('#take-survey');
+      var surveyUrl = survey.url;
+
+      if (surveyUrl.indexOf('?c=') === -1) {
+        surveyUrl += "?c=" + root.location.pathname;
+      }
+
+      $surveyLink.attr('href', surveyUrl);
+      userSurveys.trackEvent(survey.identifier, 'banner_shown', 'Banner has been shown');
+    },
+
+    setEventHandlers: function(survey) {
+      var $noThanks = $('#survey-no-thanks');
+      var $takeSurvey = $('#take-survey');
+
+      $noThanks.click(function (e) {
+        userSurveys.setSurveyTakenCookie(survey);
+        userSurveys.trackEvent(survey.identifier, 'banner_no_thanks', 'No thanks clicked');
+        e.stopPropagation();
+        return false;
+      });
+      $takeSurvey.click(function () {
+        userSurveys.setSurveyTakenCookie(survey);
+        userSurveys.trackEvent(survey.identifier, 'banner_taken', 'User taken survey');
+      });
+    },
+
+    isSurveyToBeDisplayed: function(survey) {
+      if (userSurveys.otherNotificationVisible() ||
+          GOVUK.cookie(userSurveys.surveyTakenCookieName(survey)) === 'true') {
+        return false;
+      } else if (userSurveys.userCompletedTransaction()) {
+        // We don't want any survey appearing for users who have completed a
+        // transaction as they may complete the survey with the department's
+        // transaction in mind as opposed to the GOV.UK content.
+        return false;
+      } else if ($('#user-satisfaction-survey-container').length <= 0) {
```

Comment:
@kalleth and I had a chat about this - since the content of all the upcoming surveys are related to user satisfaction, we thought that this ID name is OK as it is.